### PR TITLE
Add KaraokeType

### PR DESCRIPTION
**Pull request by memetrollsXD**
Add a website

### CODE REVIEW

File: websites/K/KaraokeType/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "id": "231774635476254721"
+  },
+  "url": "memetrolls.net",
+  "description": {
+    "en": "A simple rythm game where you type in sync with the lyrics on the screen and the singer"
+  },
+  "service": "KaraokeType",
+  "version": "1.0.0",
+  "logo": "https://memetrolls.net/etc/KaraokeType/karaoketype.png",
+  "thumbnail": "https://memetrolls.net/etc/KaraokeType/karaoketype.png",
```
Comment (Bas950) [inline websites/K/KaraokeType/metadata.json:14]: upload to imgur

File: websites/K/KaraokeType/presence.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const presence = new Presence({
+  clientId: "801765062997180437"
+});
+
+let currentURL = new URL(document.location.href),
+  currentPath = currentURL.pathname.slice(1).split("/");
+const browsingStamp = Math.floor(Date.now() / 1000);
+let presenceData: PresenceData = {
```
Comment (Bas950) [inline websites/K/KaraokeType/presence.ts:8]: merge the consts and lets

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
image
huh it isn't even website that you I think wanted to do lol

**Comment by memetrollsXD** [comment]
It's supposed to be https://memetrolls.net/etc/KaraokeType, not just "memetrolls.net"

**Comment by ririxidev** [comment]
Ehhh, but yeah. still. This domain is for minecraft server and is not even related to it. So I'm saying no to it and big sorry.

**Pull request closed without merging**